src/python/py_generate_pipeline.cpp

```diff
@@ -11,7 +11,10 @@
 #include <openvino/runtime/auto/properties.hpp>
 #include "../cpp/src/tokenizers_path.hpp"
 
+#include "./utils.hpp"
```

Comment:
```suggestion
#include "utils.hpp"
```
just add target_include dirs to current folder in cmake

Reply:
Naively adding `target_include_dirs` didn't work. It compiles successfully, but python code couldn't find symbol in `utils.cpp`.  I'll add todo to address that.
```cmake
pybind11_add_module(py_generate_pipeline py_generate_pipeline.cpp)
target_include_directories(py_generate_pipeline PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/")
```